sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -915,6 +928,27 @@ private static void validateResource(Resource resource) {
         return headers;
     }
 
+    static Map<String, String> addPreferHeader(String preferHeaderName, String preferHeaderValue, Map<String, String> headers) {
+        if (headers == null) {
+            headers = new HashMap<>();
+        }
+        return addPreferHeader(preferHeaderName + "=" + preferHeaderValue, headers);
```

Comment:
given that currently the only preferred header is minimal response, I wonder if we can avoid string concatination here:
instead of
```java
preferHeaderName + "=" + preferHeaderValue
```
we just pass `minimanRespose=true`

Reply:
This doesn't get called. The other overload gets called :) 
But since the only preferred header is minimalResponse, I will remove this overload. 
Also, just to clarify, it is passed as: `prefer:return=minimal` :)